src/main/java/com/gluonhq/substrate/target/DarwinTargetConfiguration.java

```diff
@@ -86,9 +110,8 @@ String getAdditionalSourceFileLocation() {
                 .collect(Collectors.toList());
     }
 
-    private static final List<String> macoslibs = Arrays.asList("-lffi",
-            "-lpthread", "-lz", "-ldl", "-lstrictmath", "-llibchelper",
-            "-ljava", "-lnio", "-lzip", "-ljvm", "-lobjc",
+    private static final List<String> macoslibs = Arrays.asList(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/gluonhq/substrate/pullRequest?prid=4683406)